lighthouse-core/test/report/html/renderer/util-test.js

```diff
@@ -118,7 +118,7 @@ describe('util helpers', () => {
     });
 
     // eslint-disable-next-line max-len
-    assert.equal(descriptions.networkThrottling, '565\xa0ms HTTP RTT, 1,400\xa0Kbps down, 600\xa0Kbps up (DevTools)');
+    assert.equal(descriptions.networkThrottling, 'ECT 4G - 565\xa0ms HTTP RTT, 1,400\xa0Kbps down, 600\xa0Kbps up (DevTools)');
```

Comment:
a tad concerned sophisticated folks will look at this, and then look at the spec, and be all 🧐

but hopefully that just means they conclude the spec should be fixed just we do? 😆